source/reference/minio-mc-admin/mc-admin-scanner-status.rst

```diff
@@ -0,0 +1,129 @@
+-----------
+
+.. start-mc-admin-scanner-status-desc
+
+The :mc-cmd:`mc admin scanner status` command displays a real-time summary of :ref:`scanner <minio-concepts-scanner>` information for a MinIO Server.
+
+.. end-mc-admin-scanner-status-desc
+
+This command has also be run by using ``mc admin scanner info``.
+
+.. admonition:: Use ``mc admin`` on MinIO Deployments Only
+   :class: note
+
+   .. include:: /includes/facts-mc-admin.rst
+      :start-after: start-minio-only
+      :end-before: end-minio-only
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following example returns information about the current state of the scanner process.
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc admin scanner status myminio
+
+      The command returns results similar to the following:
+
+      .. code-block:: shell
+         
+         Overall Statistics                               	
+         ------------------                                                  	
+         Last full scan time:   0d0h15m; Estimated 2879.79/month             	
+         Current cycle:         (between cycles)                             	
+         Active drives: 0                                                    	
+                                                                             	
+         Last Minute Statistics                                              	
+         ----------------------                                              	
+         Objects Scanned:       3 objects; Avg: 67.611µs; Rate: 4320/day     	
+         Versions Scanned:      3 versions; Avg: 2.506µs; Rate: 4320/day     	
+         Versions Heal Checked: 0 versions; Avg: 0ms                         	
+         Read Metadata:         3 objects; Avg: 40.817µs, Size: 395 bytes/obj	
+         ILM checks:            3 versions; Avg: 714ns                       	
+         Check Replication:     3 versions; Avg: 892ns                       	
+         Verify Deleted:        0 folders; Avg: 0ms                          	
+         Yield:                 18ms total; Avg: 6ms/obj     
+
+   .. tab-item:: SYNTAX
+
+      The command has the following syntax:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc admin scanner status ALIAS
+                                [--bucket <string>]     \
+                                [--interval <value>]   \
+                                [--max-paths <value>]  \
+                                [-n <integer>]         \
+                                [--nodes <string>] 
+
+      .. include:: /includes/common-minio-mc.rst
+         :start-after: start-minio-syntax
+         :end-before: end-minio-syntax
+
+Parameters
+~~~~~~~~~~
+
+.. mc-cmd:: ALIAS
+   :required:
+
+   The :ref:`alias <alias>` of the MinIO deployment for which to display :ref:`scanner <minio-concepts-scanner>` API operations.
+
+.. mc-cmd:: --bucket
+   :optional:
+
+   Show information about the scanning of a specified bucket.
```

Comment:
```suggestion
   Show information about the scanning of a specified bucket and exit.
```

Reply:
Not needed by a different edit to this line.